README.md

```diff
@@ -190,6 +190,59 @@ Roses are red
 Violets are blue
 ```
 
+### Text Document Parsing
+
+The `partition_text` function within `unstructured` can be used to parse simple
+text files into elements.
+
+`partition_text` accepts filenames, file-like object, raw text, and a list
+of string objects as input. The following four snippets are for parsing text files:
```

Comment:
I'd add a comment about what each element in the list should be if you pass in a list of strings. Is each element a paragraph?

Reply:
Decided to remove that option

Reply:
👍